From 9b1bcfd74b81e856754fa725fce3705405e0f915 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 5 Apr 2016 16:36:48 +0000
Subject: [PATCH 0663/1000] MANTIS 0040742 extension: Tell ConveyorDev about
 CUSS session end.

ConveyorDev.idl
V1.10
added CMD_SESSION_BEGIN, CMD_SESSION_END

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1000 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index 840e0537..2703944a 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.9";
+   const string interface_version = "1.10";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -55,6 +55,7 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
     with cmd: DeviceComponents::command
       with cmd.type:   conveyordev::commandType
       with cmd.data:   conveyordev::bagInfo                 for CMD_DELIVERY_NOTE_SET, CMD_PIECE_RFID_WRITE
+      with cmd.data:   string                               for CMD_SESSION_BEGIN       
       with cmd.data:   DeviceComponents::nullData           for the others
 */
 
@@ -390,7 +391,13 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       CMD_VERIFICATION_PROCESS,              /**< read properties of the baggage on verification belt  */
       CMD_PARKING_FORWARD,                   /**< transport a piece from parking belt to main belt  */
       CMD_PARKING_PROCESS,                   /**< read properties of the baggage on parking belt  */
-      CMD_PARKING_RETURN                     /**< transport a piece from parking belt to verification belt  */
+      CMD_PARKING_RETURN,                    /**< transport a piece from parking belt to verification belt  */
+      CMD_SESSION_BEGIN,                     /**< start a session, 
+                                                  this command can be called parallel to other commands, 
+                                                  no response indication will be sent (RC_ACCEPTED is not allowed) */
+      CMD_SESSION_END                        /**< end of the current session,
+                                                  this command can be called parallel to other commands, 
+                                                  no response indication will be sent (RC_ACCEPTED is not allowed) */
    };
 
    /**
-- 
2.41.0.windows.1

